OrbitCore/CoreApp.cpp

```diff
@@ -2,12 +2,66 @@
 // Copyright Pierric Gimmig 2013-2017
 //-----------------------------------
 #include "CoreApp.h"
+
+#include "Capture.h"
+#include "ConnectionManager.h"
+#include "OrbitBase/Logging.h"
 #include "TcpClient.h"
 
 CoreApp* GCoreApp;
 
 void CoreApp::InitializeManagers() {
   transaction_manager_ = std::make_unique<orbit::TransactionManager>(
-    GTcpClient.get(), GTcpServer.get());
+      GTcpClient.get(), GTcpServer.get());
   symbols_manager_ = std::make_unique<orbit::SymbolsManager>(GCoreApp);
+
+  SetupMemoryTransaction();
+
+  is_client_ = ConnectionManager::Get().IsClient();
+  is_service_ = ConnectionManager::Get().IsService();
+}
+
+void CoreApp::GetRemoteMemory(
+    uint32_t pid, uint64_t address, uint64_t size,
+    std::function<void(std::vector<byte>&)> callback) {
```

Comment:
I'm not a big fan of this "byte" typedef instead of just char or uint8_t.

Reply:
I agree, I switched to uint8_t.